client/src/contexts/UserProvider.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import UserContext from './UserContext';
+import PropTypes from 'prop-types';
+import Logout from '../pages/Logout';
+
+const UserProvider = (props) => {
+  const token = localStorage.getItem('loggedInUserToken');
+  try {
+    let base64Url = token.split('.')[1];
+    let base64 = base64Url.replace('-', '+').replace('_', '/');
+    let decodedData = JSON.parse(
+      Buffer.from(base64, 'base64').toString('binary')
+    );
+    const loggedInUserId = decodedData.userId;
+
+    return (
+      <UserContext.Provider value={[loggedInUserId]}>
```

Comment:
```suggestion
      <UserContext.Provider value={loggedInUserId}>
```

Reply:
If I leave out the brackets, the patch will no longer work. It seems to take only the first digit of the Id?
`users.js:59 PATCH http://localhost:3000/api/users/5 500 (Internal Server Error)`